canonical-websites/www.ubuntu.com
# Amendments re the use of the Kubernetes Trademark

## Done

Amendments re the use of the Kubernetes Trademark

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at
  - [http://0.0.0.0:8001/kubernetes](http://0.0.0.0:8001/kubernetes)
  - [http://0.0.0.0:8001/kubernetes/managed](http://0.0.0.0:8001/kubernetes/managed)
  - [http://0.0.0.0:8001/cloud](http://0.0.0.0:8001/cloud)
- Check the the changes made correspond with the changes requested in the document

**NB: This doesn't include the footnote asterix changes requested** 

## Issue / Card

Fixes #2469 

### Docs

- [/kubernetes](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.tlzom1wfx0gd)
- [/kubernetes/managed](https://docs.google.com/document/d/1viNQgt99D2_b873Yd98iNT5vEzIbPQFgNI5XBFnszHk/edit#)
- [/cloud](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit#)
